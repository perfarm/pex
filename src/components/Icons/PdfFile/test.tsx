import React from 'react';

import { render } from '@testing-library/react';

import { PdfFile } from '.';

describe('PdfFile', () => {
  test('should match snapshot', () => {
    const { container } = render(<PdfFile />);

    expect(container).toMatchSnapshot();
  });
});
