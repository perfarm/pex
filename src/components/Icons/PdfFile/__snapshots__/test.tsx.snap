// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PdfFile should match snapshot 1`] = `
<div>
  <svg
    class="PJLV"
    fill="none"
    height="18"
    viewBox="0 0 20 20"
    width="18"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      class="PJLV"
      clip-rule="evenodd"
      d="M2.5049 2.7866C2.46228 2.7866 2.42142 2.80353 2.39128 2.83366C2.36115 2.8638 2.34422 2.90467 2.34422 2.94728V12.108C2.34422 12.4792 2.04331 12.7801 1.67211 12.7801C1.30091 12.7801 1 12.4792 1 12.108V2.94728C1 2.54816 1.15855 2.16538 1.44077 1.88316C1.723 1.60093 2.10577 1.44238 2.5049 1.44238H12.4984C12.6766 1.44238 12.8476 1.51319 12.9736 1.63924L18.8031 7.46876C18.9292 7.59481 19 7.76576 19 7.94401V12.108C19 12.4792 18.6991 12.7801 18.3279 12.7801C17.9567 12.7801 17.6558 12.4792 17.6558 12.108V8.61612H12.4984C12.1272 8.61612 11.8263 8.31521 11.8263 7.94401V2.7866H2.5049ZM13.1705 3.73711L16.7053 7.2719H13.1705V3.73711Z"
      fill="#898989"
      fill-rule="evenodd"
    />
    <path
      class="PJLV"
      d="M2.86032 16.8546H1.84307C1.73262 16.8546 1.64307 16.765 1.64307 16.6546V16.2868C1.64307 16.1763 1.73262 16.0868 1.84307 16.0868H2.86032C3.04843 16.0868 3.20154 16.0562 3.31966 15.9949C3.43777 15.9315 3.52417 15.844 3.57885 15.7325C3.63354 15.6209 3.66088 15.4951 3.66088 15.3551C3.66088 15.213 3.63354 15.0806 3.57885 14.9581C3.52417 14.8357 3.43777 14.7372 3.31966 14.6629C3.20154 14.5885 3.04843 14.5513 2.86032 14.5513H2.1843C2.07384 14.5513 1.9843 14.6408 1.9843 14.7513V18.3574C1.9843 18.4678 1.89475 18.5574 1.7843 18.5574H1.2C1.08954 18.5574 1 18.4678 1 18.3574V13.9803C1 13.8698 1.08954 13.7803 1.2 13.7803H2.86032C3.23435 13.7803 3.55479 13.8481 3.82165 13.9837C4.09069 14.1171 4.29629 14.3019 4.43847 14.5382C4.58065 14.7744 4.65173 15.0445 4.65173 15.3486C4.65173 15.657 4.58065 15.9238 4.43847 16.1491C4.29629 16.3744 4.09069 16.5483 3.82165 16.6708C3.55479 16.7933 3.23435 16.8546 2.86032 16.8546Z"
      fill="#898989"
    />
    <path
      class="PJLV"
      d="M9.88845 18.5574H9.05009C8.93897 18.5574 8.84915 18.4668 8.8501 18.3557L8.85324 17.9879C8.85418 17.8781 8.94345 17.7896 9.05324 17.7896H9.88845C10.1466 17.7896 10.3642 17.7317 10.5414 17.6157C10.7185 17.4976 10.852 17.3259 10.9416 17.1006C11.0335 16.8753 11.0794 16.603 11.0794 16.2837V16.0507C11.0794 15.8057 11.0532 15.5903 11.0007 15.4044C10.9504 15.2184 10.8749 15.062 10.7743 14.9352C10.6737 14.8083 10.5501 14.7132 10.4036 14.6497C10.257 14.5841 10.0886 14.5513 9.89829 14.5513H9.02869C8.91823 14.5513 8.82869 14.4618 8.82869 14.3513V13.9803C8.82869 13.8698 8.91823 13.7803 9.02869 13.7803H9.89829C10.2176 13.7803 10.5096 13.835 10.7743 13.9443C11.0412 14.0515 11.2719 14.2057 11.4666 14.4069C11.6613 14.6082 11.8111 14.8488 11.9161 15.1288C12.0233 15.4065 12.0769 15.7161 12.0769 16.0573V16.2837C12.0769 16.6227 12.0233 16.9322 11.9161 17.2122C11.8111 17.4922 11.6613 17.7328 11.4666 17.934C11.2741 18.133 11.0434 18.2873 10.7743 18.3966C10.5075 18.5038 10.2122 18.5574 9.88845 18.5574ZM9.20614 13.7803C9.3166 13.7803 9.40614 13.8698 9.40614 13.9803V18.3574C9.40614 18.4678 9.3166 18.5574 9.20614 18.5574H8.62185C8.51139 18.5574 8.42185 18.4678 8.42185 18.3574V13.9803C8.42185 13.8698 8.51139 13.7803 8.62185 13.7803H9.20614Z"
      fill="#898989"
    />
    <path
      class="PJLV"
      d="M16.6674 13.7803C16.7778 13.7803 16.8674 13.8698 16.8674 13.9803V18.3574C16.8674 18.4678 16.7778 18.5574 16.6674 18.5574H16.0831C15.9726 18.5574 15.8831 18.4678 15.8831 18.3574V13.9803C15.8831 13.8698 15.9726 13.7803 16.0831 13.7803H16.6674ZM18.5703 15.821C18.6808 15.821 18.7703 15.9106 18.7703 16.021V16.3888C18.7703 16.4993 18.6808 16.5888 18.5703 16.5888H16.7983C16.6879 16.5888 16.5983 16.4993 16.5983 16.3888V16.021C16.5983 15.9106 16.6879 15.821 16.7983 15.821H18.5703ZM18.8 13.7803C18.9105 13.7803 19 13.8698 19 13.9803V14.3513C19 14.4618 18.9105 14.5513 18.8 14.5513H16.7983C16.6879 14.5513 16.5983 14.4618 16.5983 14.3513V13.9803C16.5983 13.8698 16.6879 13.7803 16.7983 13.7803H18.8Z"
      fill="#898989"
    />
  </svg>
</div>
`;
